flink-staging/flink-language-binding/flink-python/src/test/python/org/apache/flink/languagebinding/api/python/flink/test/test_custom.py

```diff
@@ -0,0 +1,76 @@
+# ###############################################################################
```

Comment:
The name of this file could be a bit more explanatory like: "test_custom_type_serializer"